apps/scully-docs/src/app/landing/landing.component.html

```diff
@@ -109,20 +109,22 @@ <h3>TOOLS & RESOURCES</h3>
 </div>
 
 <div class="container-tools">
-  <div class="guide-1" [routerLink]="'/docs/getting-started'">
-    <app-book
-      [header]="'Guides'"
-      [text]="'Easy step by step guides to help get you started.'"
-    >
-    </app-book>
-  </div>
-  <div class="guide-2" [routerLink]="'/docs/blog'">
+  <div class="guide-1">
     <app-book
+      [href]="'docs/blog/'"
       [header]="'Create a Blog'"
       [text]="
         'Easy step by step to create your own blog in less than 5 minutes.'
       "
     >
     </app-book>
   </div>
+  <div class="guide-2">
+    <app-book
```

Comment:
Why did you move the Getting Started to second position? Most people click that one. Shouldn't it be first?